src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -142,6 +143,18 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
     );
   };
 
+  const CreateWorkPackageButton = () => (
+    <MenuItem
+      onClick={() => history.push(`${routes.CHANGE_REQUESTS_NEW}?wbsNum=${projectWbsPipe(project.wbsNum)}&createWP=${true}`)}
+      disabled={!isLeadership(user.role)}
```

Comment:
Should only be disabled for guests, since a non-leadership user could just click the "Request Change" button and fill in the info manually anyway